plugins/poll/assets/stylesheets/common/poll.scss

```
@@ -25,9 +25,16 @@ div.poll {
   }
 
   img {
+    // TODO: remove once disable_image_size_calculations is removed
     // needed to override internal styles in image-sizing hack
     max-width: 100% !important;
     height: auto;
+    // Hacky way to stop images without width/height
+    // from causing abrupt unintended scrolling
+    &:not([width]):not([height]) {
```

This will select elements where the width is missing AND the height is missing.

I think we should select elements where the width is missing OR the height is missing. The browser needs both in order to calculate the aspect ratio.

```suggestion
    &:not([width]), &:not([height]) {
```